src/main/java/racinggame/view/InputView.java

```diff
@@ -0,0 +1,18 @@
+package racinggame.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
`static final` 키워드를 사용하여 상수로 선언하셨으니, 컨벤션에 따라 이름을 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```